src/component/admin/auth/simple-auth.tsx

```diff
@@ -0,0 +1,96 @@
+import React, { useState, useContext, useEffect } from 'react';
+import {
+    Button,
+    FormControlLabel,
+    Grid,
+    Switch,
+} from '@material-ui/core';
+import { Alert } from '@material-ui/lab';
+import PageContent from '../../common/PageContent/PageContent';
+import AccessContext from '../../../contexts/AccessContext';
+import { ADMIN } from '../../providers/AccessProvider/permissions';
+import useAuthSettings from '../../../hooks/api/getters/useAuthSettings/useAuthSettings';
+import useAuthSettingsApi, {ISimpleAuthSettings } from '../../../hooks/api/actions/useAuthSettingsApi/useAuthSettingsApi';
+
+function SimpleAuth() {
+
+    const { config } = useAuthSettings('simple');
+    const [disabled, setDisabled] = useState<boolean>(false);
+    const [info, setInfo] = useState<string>();
+    const { updateSettings, errors } = useAuthSettingsApi<ISimpleAuthSettings>('simple')
+    const { hasAccess } = useContext(AccessContext);
+
+
+    useEffect(() => {
+        setDisabled(!!config.disabled);
+    }, [ config.disabled ]);
+
+    if (!hasAccess(ADMIN)) {
+        return (
+            <Alert severity="error">
+                You need to be a root admin to access this section.
+            </Alert>
+        );
+    }
+
+    const updateDisabled = () => {
+        setDisabled(!disabled);
+    };
+
+    
+    const onSubmit = async e => {
+        e.preventDefault();
+        
+        setInfo('...saving');
+        try {
+            const settings: ISimpleAuthSettings = { disabled };
+            await updateSettings(settings);
+            setInfo('Settings stored');
```

Comment:
Add a setToastData here using the useToast to triger a toast on successful update?

Reply:
done.